### ISSUE TITLE
1.09b support

### ISSUE THREAD

**Issue by sharpwg1337** [bayaraa/d2gl#162]
Will 1.09B support ever be added?

**Comment by bibelwort** [comment]
I totally agree with this request. It would be very great if you will add support for 1.09b patch.
As far as I know 1.09d has broken CtC, e.g. a Sorc with an Occy will never teleport when hit. The 1.09b version doesn't have this issue.
Maybe there is a chance that D2GL already perfectly supports 1.09b, and you need just allow to run D2GL with this version.

And speaking of the "wrong version" message that prevents to run D2GL with officially unsupported patches. Maybe it is better to throw a warning instead of the error?
I mean, maybe some patches do not differ much from ones that you officially support (in terms of graphics and around), so we could try to run d2gl over them without any issues, but for now we cant because of the error message.

We would be very happy if you could think about it. Thank you.

**Comment by eleriaqueen** [comment]
> Maybe there is a chance that D2GL already perfectly supports 1.09b, and you need just allow to run D2GL with this version.

I tried to modify the D2GL source to run if 1.09b is detected but it crashes the game so I guess It's not that easy.
